frontend/public/kubevirt/components/vm.jsx

```diff
@@ -330,6 +230,56 @@ const mapDispatchToProps = () => ({
   watchK8sList: actions.watchK8sList,
 });
 
+const ConnectedVmDetails = ({ obj: vm }) => {
+  const vmResource = getResourceKind(VirtualMachineModel, vm.metadata.name, true, vm.metadata.namespace, false);
+  const vmiResources = getResourceKind(VirtualMachineInstanceModel, vm.metadata.name, true, vm.metadata.namespace, true, getLabelMatcher(vm));
+  const podResources = getResourceKind(PodModel, undefined, true, vm.metadata.namespace, true, getLabelMatcher(vm));
+  const migrationResources = getResourceKind(VirtualMachineInstanceMigrationModel, undefined, true, vm.metadata.namespace, false); // https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachineinstancemigration
+  const resources = [vmResource, vmiResources, podResources, migrationResources];
+
+  return (
+    <Firehose resources={resources} flatten={getFlattenForKind(VirtualMachineModel.kind)}>
+      <VmDetails_ vm={vm} />
+    </Firehose>
+  );
+};
+
+const VmDetails_ = props => {
+  const { flatten, loaded, vm, resources } = props;
+  if (loaded){
+    const vm = flatten(props.resources);
+
+    if (vm) {
+      const namespaceResourceLink = (
+        <ResourceLink kind={NamespaceModel.kind} name={vm.metadata.namespace} title={vm.metadata.namespace} />
+      );
+
+      const podResourceLink = (
+        <FirehoseResourceLink
+          {...props}
+          flatten={getFlattenForKind(PodModel.kind)}
+          filter={data => findPod(data, vm.metadata.name, VIRT_LAUNCHER_POD_PREFIX)}
+        />);
+
+      return (
+        <VmDetails
+          {...props}
+          vm={vm}
+          ResourceLink={ResourceLink}
+          NodeLink={NodeLink}
+          NamespaceResourceLink={namespaceResourceLink}
+          PodResourceLink={podResourceLink}
+          launcherPod={getPod(vm, resources, VIRT_LAUNCHER_POD_PREFIX)}
```

Comment:
use `findPod` instead of `getPod` based on previous comments (applies on other occurrences too)